thermalnetwork/network.py

```diff
@@ -670,15 +674,17 @@ def update_sys_params(self, system_parameter_path: Path, output_directory_path:
         # Load the existing system parameters
         sys_params: dict = json.loads(system_parameter_path.read_text())
         ghe_params: list = sys_params["district_system"]["fifth_generation"]["ghe_parameters"]["ghe_specific_params"]
-        for ghe_id in output_directory_path.iterdir():
-            # each ghe_id is a directory
-            if not ghe_id.is_dir():
-                continue
-            summary_data_path = ghe_id / "SimulationSummary.json"
+        for ghe in ghe_params:
+            ghe_id = ghe["ghe_id"]
+            summary_data_path = output_directory_path / ghe_id / "SimulationSummary.json"
+
+            if not summary_data_path.exists():
+                logger.error(f"Error sizing GHE: {ghe_id}")
+
```

Comment:
@vtnate I saw a comment in my inbox about this, but don't see anything here. Anyway, feel free to modify this if needed, but something was definitely wrong with it.

Reply:
I re-thought my comment and deleted it 😆 .